### PR TITLE
Add ability to specify preferred port

### LINKED ISSUE #6: Default port?
Any interest in accepting a PR to support passing in a preferred/default port?

### PR DESCRIPTION
**Pull request by djmadeira**
Fixes #6

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -1,8 +1,40 @@
...
 	t.is(typeof port, 'number');
 	t.true(port > 0);
+
+	const server = net.createServer();
+
+	await new Promise(resolve => {
+		server.listen(port, resolve);
+	});
+
+	t.true(server.address().port === port);
```
Comment (sindresorhus) [inline test.js]: Use `t.is()`

File: test.js [hunk 2]
```diff
@@ -1,8 +1,40 @@
...
+	const server = net.createServer();
+
+	await new Promise(resolve => {
+		server.listen(desiredPort, resolve);
+	});
+
+	const port = await m(desiredPort);
+	t.is(typeof port, 'number');
+	t.true(port > 0);
+	t.true(port !== desiredPort);
```
Comment (sindresorhus) [inline test.js]: Use `t.not()`

**Comment by djmadeira** [comment]
@sindresorhus @floatdrop addressed comments.

**Comment by sindresorhus** [comment]
Thank you @djmadeira :)

**Pull request merged**